### PR TITLE
 Fix #69: Updated the sketch 'catena4460_bsec_ulp' to support latest BSEC library 

### LINKED ISSUE #69: BSEC library has been updated - Need update for catena4460_bsec_ulp.ino
Bosch has updated the library for BSEC and the latest version is v1.4.7.2. 

Sketch **catena4460_bsec_ulp.ino** has been written based on BSEC software v1.4.6.0, so the sketch has to be updated to work with BSEC v1.4.7.2. Please find the code part need to be updated below:

https://github.com/mcci-catena/Catena-Sketches/blob/01c3a841b3f9fbc82fcdded2f0023268bac770f3/catena4460_bsec_ulp/catena4460_bsec_ulp.ino#L318-L327

Here in the sensor list, Bosch have changed output (virtual) sensor `BSEC_OUTPUT_IAQ_ESTIMATE` to `BSEC_OUTPUT_IAQ` in their library **bsec_datatypes.h** under "BSEC_1.4.7.2_Generic_Release_20190122\Arduino\BSEC\src\inc\". Also they have added three other output (virtual) sensors in their library:

```
BSEC_OUTPUT_STATIC_IAQ = 2,                             /*!< Unscaled indoor-air-quality estimate */ 
BSEC_OUTPUT_CO2_EQUIVALENT = 3,                         /*!< co2 equivalent estimate [ppm] */   
BSEC_OUTPUT_BREATH_VOC_EQUIVALENT = 4,                  /*!< breath VOC concentration estimate [ppm] */
``` 

Link for BSEC software: https://www.bosch-sensortec.com/bst/products/all_products/bsec#

### PR DESCRIPTION
**Pull request by dhineshkumarmcci**
Sketch catena4460_bsec_ulp.ino has been updated to support the latest version of BSEC library.

### CODE REVIEW

**Comment by dhineshkumarmcci** [comment]
Have updated the catena4460_bsec_ulp sketch with the latest base.

File: catena4460_bsec_ulp/catena4460_bsec_ulp.ino [hunk 1]
```diff
@@ -44,37 +44,41 @@ Author:
 
 using namespace McciCatena;
 
-#define STATE_SAVE_PERIOD  UINT32_C(3 * 60 * 1000) // 3 minutes: basically any change.
+constexpr uint32_t STATE_SAVE_PERIOD = UINT32_C(3 * 60 * 1000); // 3 minutes: basically any change.
```
Comment (terrillmoore) [inline catena4460_bsec_ulp/catena4460_bsec_ulp.ino]: You don't need UINT32_C() after you make it a `constexpr uint32_t`.

File: catena4460_bsec_ulp/catena4460_bsec_ulp.ino [hunk 2]
```diff
@@ -101,7 +105,7 @@ void sensorJob_cb(osjob_t *pJob);
 |
 \****************************************************************************/
 
-static const char sVersion[] = "0.2.1";
+static const char sVersion[] = "0.2.2";
```
Comment (terrillmoore) [inline catena4460_bsec_ulp/catena4460_bsec_ulp.ino]: This is at least 0.3.0.  See discussion of semantic versions at https://semver.com

File: catena4460_bsec_ulp/catena4460_bsec_ulp.ino (hunk 1 above)
Comment (dhineshkumarmcci) [reply catena4460_bsec_ulp/catena4460_bsec_ulp.ino]: Suggested changes have been done.

File: catena4460_bsec_ulp/catena4460_bsec_ulp.ino (hunk 2 above)
Comment (dhineshkumarmcci) [reply catena4460_bsec_ulp/catena4460_bsec_ulp.ino]: Changed the version to 0.3.0.

**Pull request merged**